### ISSUE TITLE
Drupal site name is set as the intended site name but wrapped in quotes when running `drush si` from an event or from the `run` build step

### ISSUE THREAD

**Issue by philwolstenholme** [lando/lando#1926]
Hi, I'm using v3.0.0-rc.23 on MacOS High Sierra 10.13.6. I've found Lando very useful, so thank you very much for your work on it.

I have a Drupal 8 environment and am trying to run the `drush si` (site install) command when my Lando instance is created.

Here's my Lando file:

```yml
name: heroheadertest
recipe: drupal8
config:
  webroot: www
services:
  appserver:
    type: php
    overrides:
      environment:
        PROJECT_KEY: 'drupal8'
        DB_USER: 'drupal8'
        DB_PASS: 'drupal8'
        DB_HOST: 'database'
        DB:PORT: '3306'
        DRUSH_OPTIONS_URI: 'https://heroheadertest.lndo.site'
    run:
      - composer global require hirak/prestissimo
      - composer install
      - ln -sf ../../../docker/config/sites/default/settings.local.php www/sites/default/settings.local.php
      - ln -sf ../../docker/config/sites/services.local.yml www/sites/services.local.yml
      - drush si standard --site-name="Example Drupal site" --account-name=root --account-pass=password --verbose -y
```

When my site finishes installing it has the Drupal site name of `"Example Drupal site"`, so the quotes have been passed through as part of the string.

If I do a ` lando ssh -s appserver` and run `drush si standard --site-name="Example Drupal site" --account-name=root --account-pass=password --verbose -y` to bypass Lando, then my site gets the name `Example Drupal site` (no extra quotes this time).

How can I escape/alter my Drush command to avoid the extra quotes reaching Drupal?

**Comment by tormi** [comment]
Just a sidenote - Lando already has `hirak/prestissimo` installed, https://github.com/lando/lando/blob/master/plugins/lando-services/services/php/7.2-apache/Dockerfile#L76 ;)

**Comment by pirog** [comment]
Our upstream parser does not want to parse this the right way so against my better judgment i added some logic in lando to handle it
https://www.youtube.com/watch?v=HKK4KmDlj8U

should be in the next release

**Comment by mstrelan** [comment]
I think you were right about this biting you in the ass.

I came across this problem today trying to pass extra parameters to `pg_dump` and came up with this workaround before finding this issue:

    tooling:
      pgdump:
        - bash -c 'EXTRA_DUMP="-Fc -O";drush sql-dump --extra-dump="$EXTRA_DUMP" --gzip --result-file=/tmp/db.sql --debug'

This was working on previous release. Updated to 3.0.0 and now that's giving me this:

    bash: -c: line 0: unexpected EOF while looking for matching `"'
    bash: -c: line 1: syntax error: unexpected end of file

Good news is that I can now remove my workaround.

Meanwhile, is there a way to pass these parameters to `lando sql-export`?
